packages/block-library/src/navigation/edit/inner-blocks.js

```diff
@@ -94,7 +83,7 @@ export default function NavigationInnerBlocks( {
 			onChange,
 			prioritizedInserterBlocks: PRIORITIZED_INSERTER_BLOCKS,
 			defaultBlock: DEFAULT_BLOCK,
-			directInsert: shouldDirectInsert,
+			directInsert: true,
```

Comment:
Is it ever false? I mean do we still need this? 🤔 seems we always will direct insert with this UX flow.

Reply:
The default value is false, so we do need it.